index.html

```diff
@@ -0,0 +1,733 @@
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+            <div class="news-container">
+                <img src="img/latest-deals.png" alt="latest deals image">
+                <div>
+                <p>Graphic novels are the ideal medium to get through these troubled times </p>
+                <p class="gray fsmall"><a href="">Callum Booth</a> . 26 minutes ago</p>
+                </div>
+            </div>
+        </section>
+    </div>
+
+        <div class="article2">
+            <ul class="article2-sub1 orange-headers">
+                <li>Latest funding rounds on Index.co
+                    <i class="fas fa-angle-right"></i>
+                </li>
+            </ul>
+
+            <ul class="article2-subelement">
+                <li class="one">Add info</li>
+                <li class="two">Get Update by email</li>
+                <li class="three">see more</li>
+            </ul>
+        </div>
+
+        <div class="article-container">
+            <div class="article-container-1">
+
+                <ul class="ul1">
+
+                    <li>
+                        <p>Encoded Genomics</p>
+                    </li>
+                </ul>
+
+                <ul class="ul2">
+                    <li class="company-logo">
+                        <img class="article-img" src="img/image.jpeg" alt="Company logo" />
+                    </li>      
+                         <li>
+                    <p>$135M</p>
+                    </li>
+                    <li>
+                        <p>SERIES D</p>
+                    </li>
+                </ul>
+
+
+
+                <UL class="ul3">
+                    <li>
+                        <p><i class="fas fa-tag"></i>
+                            Subcription Business, +2</p>
+                    </li>
+
+                    <li>
+                        <p><i class="fas fa-tag"></i>
+                            Index ventures, +4</p>
+                    </li>
+                </UL>
+
+                <ul class="ul4">
+                    <li>
+                        <p>Subscription Business, +2</p>
+                    </li>
+                    <li>Index ventures, +4</li>
+                </ul>
```

Comment:
- [ ] **All sizes** Please notice that the font size is smaller, it is changing the size of your containers:
![image](https://user-images.githubusercontent.com/53256139/88810744-6130eb80-d17b-11ea-9b27-b38765775f4d.png)